### PR TITLE
[rv_core_ibex] Fix ENUMASSIGN VCS error

### PR DESCRIPTION
**Pull request by msfschaffner**
Alternative fix for #11192 using an `prim_lc_sender`.

Signed-off-by: Michael Schaffner <msf@opentitan.org>

### CODE REVIEW

**Review by a-will — APPROVED** [review]
This seems fine too.

File: hw/ip/rv_core_ibex/rtl/rv_core_ibex.sv
Comment (tjaychen) [inline hw/ip/rv_core_ibex/rtl/rv_core_ibex.sv:319]: this lgtm, but wouldn't it been easier to just convert the `prim_buf` to a `prim_lc_sender` with `AsyncOn` set to 0?

**Review by tjaychen — APPROVED** [review]

**Comment by tjaychen** [comment]
the latest CI failure is because we made `jitter_en` into a mubi signal.
We need to add an exclusion, see #11157 , otherwise the csr tests writes a random value and the mubi sync detects a transient error.

My original PR this didn't show up, but seems like it is now. My PR is a bit far away from being merged, so i can make another one quickly. 

@msfschaffner

File: hw/ip/rv_core_ibex/rtl/rv_core_ibex.sv
Comment (msfschaffner) [reply hw/ip/rv_core_ibex/rtl/rv_core_ibex.sv:319]: We can do that as well - just thought we might as well absorb the flop (less code :))...

Comment (tjaychen) [reply hw/ip/rv_core_ibex/rtl/rv_core_ibex.sv:319]: o okay, no worries then.

**Comment by msfschaffner** [comment]
Ok thanks for the heads up @tjaychen.
I'll rebase once your fix has landed.

Given that this blocks overall CI, are we ok with moving this in once basic checks and private CI have passed?

**Comment by tjaychen** [comment]
> Ok thanks for the heads up @tjaychen. I'll rebase once your fix has landed.
> 
> Given that this blocks overall CI, are we ok with moving this in once basic checks and private CI have passed?

okay with me.

**Comment by msfschaffner** [comment]
Merging this in order to unblock master (the failure in Verilator CI is unrelated and due to a flaky package install).

**Pull request merged**
